ddtrace/appsec/_iast/_span_metrics.py

```diff
@@ -30,7 +30,9 @@ def _set_span_tag_iast_executed_sink(span):
 
 def get_iast_span_metrics() -> Dict:
     env = _get_iast_env()
-    return env.iast_span_metrics if env else dict()
+    if env:
```

Comment:
nit: opportunity for the walrus operator here
```suggestion
    if env := _get_iast_env():
```